docs/source/pythonapi/examples/tally-arithmetic.ipynb

```diff
@@ -0,0 +1,1512 @@
+       "      <th>0</th>\n",
+       "      <td>total</td>\n",
+       "      <td>absorption</td>\n",
+       "      <td>0.959067</td>\n",
+       "      <td>0.00517</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    nuclide       score      mean  std. dev.\n",
+       "bin                                         \n",
+       "0     total  absorption  0.959067    0.00517"
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute resonance escape probability using tally arithmetic\n",
+    "therm_abs_rate = sp.get_tally(name='therm. abs. rate')\n",
+    "res_esc = therm_abs_rate / abs_rate\n",
+    "res_esc.get_pandas_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The fast fission factor can be calculated as\n",
+    "$$\\epsilon=\\frac{\\Sigma_f\\phi}{\\Sigma_{f,thermal}\\phi}$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div style=\"max-height:1000px;max-width:1500px;overflow:auto;\">\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>nuclide</th>\n",
+       "      <th>score</th>\n",
+       "      <th>mean</th>\n",
+       "      <th>std. dev.</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>bin</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>total</td>\n",
+       "      <td>fission</td>\n",
+       "      <td>1.080124</td>\n",
+       "      <td>0.008126</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    nuclide    score      mean  std. dev.\n",
+       "bin                                      \n",
+       "0     total  fission  1.080124   0.008126"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute fast fission factor factor using tally arithmetic\n",
+    "therm_fiss_rate = sp.get_tally(name='therm. fiss. rate')\n",
+    "tot_fiss_rate = sp.get_tally(name='tot. fiss. rate')\n",
+    "fast_fiss = tot_fiss_rate / therm_fiss_rate\n",
+    "fast_fiss.get_pandas_dataframe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The thermal flux utilization is calculated as\n",
+    "$$f=\\frac{\\Sigma_{a,thermal}^F\\phi}{\\Sigma_a\\phi}$$\n",
+    "where $F$ denotes fuel."
```

Comment:
On my system this (and the alpha from the other notebook) render correctly. Hmmm